api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -614,6 +625,7 @@ COUNT(distinct p1.PERSON_ID) as count_value,COUNT(distinct p1.PERSON_ID) as sour
 from \`${BQ_PROJECT}.${BQ_DATASET}.person\` p1 inner join \`${BQ_PROJECT}.${BQ_DATASET}.drug_exposure\` co1
 on p1.person_id = co1.person_id
 where co1.drug_concept_id != co1.drug_source_concept_id
+and co1.drug_source_concept_id not in (select distinct drug_concept_id from \`${BQ_PROJECT}.${BQ_DATASET}.drug_exposure\`)
```

Comment:
I believe you can remove the above line e.g. `co1.drug_concept_id != co1.drug_source_concept_id` from most of these queries, it should be redundant with the new check you've added.